docs/client_install/src/asciidoc/index.adoc

```diff
@@ -21,7 +21,7 @@
 * @@@ END COPYRIGHT @@@ 
 ////
 
-= Trafodion Client Installation Guide
+= Client Installation Guide
```

Comment:
Because all the guides now have the Trafodion logo on them.

On Tue, Feb 23, 2016 at 12:56 PM, DaveBirdsall <notifications@github.com>
wrote:

> In docs/client_install/src/asciidoc/index.adoc
> <https://github.com/apache/incubator-trafodion/pull/338#discussion_r53839737>
> :
>
> > @@ -21,7 +21,7 @@
> >  * @@@ END COPYRIGHT @@@
> >  ////
> >
> > -= Trafodion Client Installation Guide
> > += Client Installation Guide
>
> Just relating this back to what I see on the web site. If I click here:
> http://trafodion.apache.org/docs/2.0.0/client_install/index.html, I see a
> title, "Trafodion Client Installation Guide". Is the change above going to
> change this to "Client Installation Guide"? And if so, why are we removing
> the name "Trafodion" from it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/338/files#r53839737>.
>



-- 
Thanks,

Gunnar
*If you think you can you can, if you think you can't you're right.*